ui/src/components/Rules/RuleGroup/index.tsx

```diff
@@ -47,50 +47,53 @@ const RuleGroup = (props: {
   }
 
   return (
-    <div className="relative mb-5 flex w-full flex-col overflow-hidden rounded-xl bg-zinc-800 bg-cover bg-center p-4 text-zinc-400 shadow ring-1 ring-zinc-700 sm:flex-row">
-      <div className="relative z-10 flex w-full min-w-0 flex-col pr-4 sm:w-5/6 sm:flex-row">
-        <div className="mb-3 flex flex-col sm:mb-0 sm:w-5/6">
-          <div className="flex text-xs font-medium text-white">
-            <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-base font-bold text-white sm:text-lg">
-              {props.group.name}
-            </div>
+    <div className="flex-auto">
+      <div className="text-base font-bold text-white sm:text-lg">
+        {props.group.name}
+      </div>
+      <div className="mt-2 h-28 overflow-hidden overflow-y-auto overflow-ellipsis font-semibold">
+        {props.group.description}
+      </div>
+      <div className="mb-2 mt-2 grid grid-cols-3 gap-3">
```

Comment:
Very minor change but I think the buttons look too close to the text above
```suggestion
      <div className="mb-3 mt-2 grid grid-cols-3 gap-3">
```

Reply:
I actually changed it to use the same styles and div layout as from the Collection page. Real difference is that the bottom row here are buttons instead of information like on the collection page. The button row styling and the info row styling here have different styles than the collection page, because one has three columns and the Collection page only has two (as a flex row instead of a grid). This makes the mobile layout different though, as the buttons stack here and on the collection page the info items are made into 4 columns.